src/reformat.c

```diff
@@ -407,15 +407,15 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image, avifRGBIm
                     // in the following unormU and unormV grid as follows:
                     //
                     // unorm[0][0] = closest         ( weights: bilinear: 9/16, nearest: 1 )
-                    // unorm[1][0] = adjacent row    ( weights: bilinear: 3/16, nearest: 0 )
-                    // unorm[0][1] = adjacent col    ( weights: bilinear: 3/16, nearest: 0 )
+                    // unorm[1][0] = adjacent col    ( weights: bilinear: 3/16, nearest: 0 )
+                    // unorm[0][1] = adjacent row    ( weights: bilinear: 3/16, nearest: 0 )
                     // unorm[1][1] = diagonal        ( weights: bilinear: 1/16, nearest: 0 )
                     //
                     // It then weights them according to the requested upsampling set in avifRGBImage.
 
                     uint16_t unormU[2][2], unormV[2][2];
 
-                    // How many bytes to add to a pointer index to get to the adjacent (lesser) sample in a given direction
+                    // How many bytes to add to a uint8_t pointer index to get to the adjacent (lesser) sample in a given direction
```

Comment:
The addition of "uint8_t" to this comment is intended to clarify that these values are added to ptrU8 and ptrV8, not ptrU16 and ptrV16.